File: src/main/java/com/rody/guess/GameFor.java
package com.rody.guess;

import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        for(int i=0; i<=3; i++){
            System.out.println(i);
        }
        Scanner scanner= new Scanner(System.in);
        int number= 0;
        for (int i=0; i<=5; i++){
            System.out.println("Please enter a number");
            number= scanner.nextInt();
            System.out.println("Number:"+i+number);
            if(number==-1){
                break;
            }
        }

        System.out.println("End");

    }
}

File: src/main/kotlin/com/rody/game/gamefor.kt
package com.rody.game

import java.util.*

fun main(args: Array<String>) {
    val scanner= Scanner(System.`in`)
    var number=0;
    val secret_number= Random().nextInt(10)+1
    println(secret_number)
    for( i in 1..4){
        print("Please enter a number(${i}/4)");
        number= scanner.nextInt()
        println("第${i}次 $number")

        if (number==secret_number){
            println("great")
            break;
        }
        else if(number > secret_number) {
            println("lower")
        }
        else {
            println("higher")
        }
    }
}